package.json

```diff
@@ -52,6 +52,7 @@
     "@floating-ui/react-dom-interactions": "^0.9.1",
     "classnames": "^2.3.2",
     "commitizen": "^4.2.5",
+    "flowbite-react": "^0.3.6",
```

Comment:
You don't need to include the `flowbite-react` here. So, there is no need to also update the `package-lock.json` and `yarn.lock` files. Could you please revert these changes?